app/src/components/HeaderItem.vue

```diff
@@ -1,20 +1,30 @@
-<template>
-  <v-tab :to="to" :href="href">
-    <v-icon>{{ icon }}</v-icon>
-    <span class="ml-1" :class="computedClass">
-      <slot></slot>
-    </span>
-  </v-tab>
-</template>
-
 <script>
 export default {
   name: 'headerItem',
   props: ['icon', 'isSelected', 'to', 'href'],
   computed: {
     computedClass: function() {
-      return !this.isSelected ? 'hidden-sm-and-down' : ''
+      return !this.isSelected ? 'active' : ''
     }
   }
 }
 </script>
+
```

Comment:
Although, this is a valid style, our style guide uses template, script, style in that order
For more style guides, refer to https://vuejs.org/v2/style-guide/

Reply:
> Although, this is a valid style, our style guide uses template, script, style in that order
> For more style guides, refer to https://vuejs.org/v2/style-guide/

Done.

Reply:
> Although, this is a valid style, our style guide uses template, script, style in that order
> For more style guides, refer to https://vuejs.org/v2/style-guide/

Done.